src/query/service/src/pipelines/pipeline_builder.rs

```diff
@@ -192,10 +197,62 @@ impl PipelineBuilder {
             PhysicalPlan::RuntimeFilterSource(runtime_filter_source) => {
                 self.build_runtime_filter_source(runtime_filter_source)
             }
+            PhysicalPlan::DeletePartial(delete) => self.build_delete_partial(delete),
+            PhysicalPlan::DeleteFinal(delete) => self.build_delete_final(delete),
             PhysicalPlan::RangeJoin(range_join) => self.build_range_join(range_join),
         }
     }
 
+    /// The flow of Pipeline that build_delete_partial and build_delete_final build is as follows:
+    /// +---------------+      +-----------------------+
+    /// |MutationSource1| ---> |SerializeDataTransform1|   ------
+    /// +---------------+      +-----------------------+         |      +-----------------------+      +----------+
+    /// |     ...       | ---> |          ...          |   ...   | ---> |TableMutationAggregator| ---> |CommitSink|
+    /// +---------------+      +-----------------------+         |      +-----------------------+      +----------+
+    /// |MutationSourceN| ---> |SerializeDataTransformN|   ------
+    /// +---------------+      +-----------------------+
```

Comment:
I think we should adjust this doc, `TableMutationAggreator` and `CommitSink` are not "chained" at this phase?